buildenv/jenkins/common/variables-functions

```diff
@@ -447,17 +447,20 @@ def get_date() {
 def set_test_variables() {
     set_release()
     set_jdk_folder()
-    
+
     if ( SPEC ==~ /.*valhalla.*/) {
-        JAVA_VERSION = "Valhalla"
+        JDK_VERSION = "Valhalla"
     } else {
-        JAVA_VERSION = "SE" + "${SDK_VERSION}" + "0"
+        JDK_VERSION = SDK_VERSION
     }
+    JDK_VERSION = (JDK_VERSION == 'next') ? '13' : JDK_VERSION
```

Comment:
Since PR https://github.com/eclipse/openj9/pull/4843 is merged, we do not need to set `JDK_VERSION` anymore (AUTO_DETECT is on by default).

I think we can do:

```
JDK_VERSION = (JDK_VERSION == 'next') ? '' : JDK_VERSION
```

Could you run a personal build to confirm?